language/go/generate.go

```diff
@@ -662,6 +670,12 @@ func (g *generator) setCommonAttrs(r *rule.Rule, pkgRel string, visibility []str
 	if !target.copts.isEmpty() {
 		r.SetAttr("copts", g.options(target.copts.build(), pkgRel))
 	}
+	if !target.cppopts.isEmpty() {
+		r.SetAttr("cppopts", g.options(target.cppopts.build(), pkgRel))
+	}
+	if !target.cxxopts.isEmpty() {
+		r.SetAttr("cxxopts", g.options(target.cxxopts.build(), pkgRel))
+	}
```

Comment:
Please add `cppopts` and `cxxopts` to the `MergeableAttrs` sets for the rules in kinds.go. Everyhting that has `copts` and `clinkopts` should also have these set.